include/aspect/heating_model/tidal_heating.h

```diff
@@ -74,10 +83,25 @@ namespace aspect
          * time-averaged strain rate = constant_tidal_strain_rate
          * tidal frequency = tidal_frequency
          * elastic shear modulus = elastic_shear_modulus
+         *
+         * latitudinal_variation is bool that used to decide whether latitudinal variation would be considered.
+         * If true, strain rate varies by latitude having maximum at the poles and minimum at the equator.
+         * The equation follows as (maximum_tidal_strain_rate - minimum_tidal_strain_rate)*cos(theta/2)+(maximum_tidal_strain_rate+minimum_tidal_strain_rate)/2.
+         * This is shown in Fig.3 of Nimmo et al. (2007) (https://doi.org/10.1016/j.icarus.2007.04.021).
          */
         double tidal_frequency;
         double elastic_shear_modulus;
         double constant_tidal_strain_rate;
+        /**
+         * Specify the distribution of time-averaged tidal strain rate.
+         */
+        enum CustomTidalHeating
+        {
+          constant,
+          latitudinal_variation
+        } custom_strain_rate;
```

Comment:
just for readability, please put empty lines before the comment in line 95 and after the variable in line 102 (i.e. around the customtidalheating enum)

also maybe rename the enum to `StrainRateDistribution` and the variable to `strain_rate_distribution`